dev-packages/e2e-tests/test-applications/nextjs-14/package.json

```diff
@@ -26,8 +26,20 @@
     "wait-port": "1.0.4"
   },
   "devDependencies": {
+    "@sentry-internal/feedback": "latest || *",
```

Comment:
Do we need all of these?

Reply:
I think so, at least I couldn't be bothered to filter them out. These are all deps that nextjs depends on or could ever depend on.